### PR TITLE
ISSITES-5938 clean up mobile styling for learn site

### PR DESCRIPTION
**Pull request by liaoethan**

### CODE REVIEW

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-theme-css/src/css/_documentations.scss [hunk 1]
```diff
@@ -203,7 +203,7 @@
 		padding: $spacing-xs 0.25rem $spacing-xs $spacing-md;
 
 		button {
-			color: $primary;
+			color: var(--color-action-neutral-default);
```
Comment (allen-ziegenfus) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-theme-css/src/css/_documentations.scss]: use the sass variable here

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-theme-css/src/css/_landing_page.scss [hunk 2]
```diff
@@ -5,9 +5,12 @@
 	@include media-breakpoint-down(sm) {
 		justify-content: center;
 	}
+	@include media-breakpoint-down(md) {
+		margin-top: $spacing-md;
+	}
 
 	.autofit-row-center > .autofit-col {
-		justify-content: initial;
+		display: contents;
```
Comment (allen-ziegenfus) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-theme-css/src/css/_landing_page.scss]: we can also achieve that with flex: 100%

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-theme-css/src/css/_landing_page.scss [hunk 3]
```diff
@@ -17,9 +20,9 @@
 	}
 
 	.section-card {
-		border: 1px solid rgba(9, 16, 24, 0.12);
+		border: 1px solid var(--color-neutral-3);
```
Comment (allen-ziegenfus) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-theme-css/src/css/_landing_page.scss]: use sass variable

**Review by allen-ziegenfus — CHANGES_REQUESTED** [review]

**Comment by allen-ziegenfus** [comment]
@liaoethan 

I'm a little confused by this PR - it looks like you have mixed in commits for the public site nav that were already merged into the learn branch but here they are causing merge conflicts.

Please create a new PR and send it to me on the `learn` branch rather than master

Also it's not clear to me whether we need the arrows are not on landing pages - I noticed you changed the color to black on the h4 links, but that doesn't match what I see on the mockups. 

For the arrow discussion let's see what May says on this thread: https://liferay.slack.com/archives/C03SJA564MS/p1674585812333319

Also not all these changes are really related to mobile so i'm having trouble sorting through what we are trying to accomplish with this ticket

**Comment by liaoethan** [comment]
Hey @allen-ziegenfus, I recreated the pull here and took out the changes (landing page arrows) that didn't affect the mobile styling:
https://github.com/allen-ziegenfus/liferay-portal/pull/92

**Pull request closed without merging**